Gemfile

```diff
@@ -15,14 +17,15 @@ ruby '2.5.8'
 group :development do
   gem 'ffi-icu'
   gem 'jekyll', '~>4.0'
+  gem 'jshintrb', '~>0.3.0'
   gem 'json', '>= 1.9'
-  gem 'less', '2.4.0'
```

Comment:
Can you please clarify why `less` is being dropped, here?

Reply:
I don't remember specifically, but I seem to remember that this is not being used anywhere.

Reply:
I remember now... This was moved out, as it is a dependency on another gem, according to Gemfile.lock.  So I believe it isn't a direct dependency of the work here.

Reply:
It is on line 24 of the Gemfile.